Makefile

```diff
@@ -23,48 +23,38 @@ govet:
 	@echo "Running go vet"
 	go vet
 
-.PHONY: config
-config:
-	@echo "deploying configmap"
-	kubectl create -f config/examples/sriovdp-configmap.yaml
+outdir:
+	mkdir -p _out || :
+
+.PHONY: deps-update
+deps-update:
+	go mod tidy && go mod vendor
+
+.PHONY: deps-clean
+deps-clean:
+	rm -rf vendor
+
+.PHONY: binaries
+binaries: outdir deps-update build
+
+.PHONY: clean
+clean:
+	rm -rf _out
 
 .PHONY: image
-image: build
+image: binaries
 	@echo "building image"
-	$(RUNTIME) build -f images/Dockerfile -t quay.io/$(REPOOWNER)/$(IMAGENAME):$(IMAGETAG) .
-
-.PHONY: crd
-crd:
-	@echo "deploying crd"
-	kubectl create -f manifests/crd-apiextension-v1beta1.yaml
+	$(RUNTIME) build -f Dockerfile -t quay.io/$(REPOOWNER)/$(IMAGENAME):$(IMAGETAG) .
 
 .PHONY: push
 push: image
 	@echo "pushing image"
 	$(RUNTIME) push quay.io/$(REPOOWNER)/$(IMAGENAME):$(IMAGETAG)
 
-.PHONY: deploy
-deploy: push
-	@echo "deploying Resource Topology Exporter"
-	kubectl create -f manifests/resource-topology-exporter-ds.yaml
-
-.PHONY: deploy-pod
```

Comment:
Do we not want to have a make target here for deployment of the sample pods requesting sriov devices that are in the manifest directory?